docs/registry.md

```diff
@@ -66,5 +67,43 @@ If the module is successfully registered, you can use this converter through con
 
 ```python
 converter_cfg = dict(type='Converter1', a=a_value, b=b_value)
-converter = build_converter(converter_cfg)
+converter = CONVERTERS.build(converter_cfg)
+```
+
+## Hierarchy Registry
+
+Hierarchy structure is used for similar registries from different packages.
+For example, both [MMDetection](https://github.com/open-mmlab/mmdetection) and [MMClassification](https://github.com/open-mmlab/mmclassification) have `MODEL` registry define as followed:
+In MMDetection:
+
+```python
+from mmcv.utils import Registry
+from mmcv.cnn import MODELS as MMCV_MODELS
+MODELS = Registry('model', parent=MMCV_MODELS)
+
+@MODELS.register_module()
+class NetA(nn.Module):
+    def forward(self, x):
+        return x
+```
+
+In MMClassification:
+
+```python
+from mmcv.utils import Registry
+from mmcv.cnn import MODELS as MMCV_MODELS
+MODELS = Registry('model', parent=MMCV_MODELS)
+
+@MODELS.register_module()
+class NetB(nn.Module):
+    def forward(self, x):
+        return x + 1
+```
+
+We could build either `NetA` or `NetB` by:
+
+```python
+from mmcv.cnn import MODELS as MMCV_MODELS
+net_a = MMCV_MODELS.build(cfg=dict(type='mmdet.NetA'))
```

Comment:
There are two questions that might need to be answered  in the doc. 

1. In this case, can we still build net_a and net_b through

```python
net_a = MMCV_MODELS.build(cfg=dict(type='NetA'))
net_b = MMCV_MODELS.build(cfg=dict(type='NetB'))
```
2. If MMCls and MMDet both create a class named `NetA`, what will happen if we try

```python
net_a = MMCV_MODELS.build(cfg=dict(type='NetA'))
```
Does this depend on the order during registration or something else? Or the conflict can only be resolved by use `mmdet.NetA` and `mmcls.NetA` ?

Reply:
Currently, it is depend on the order.